tasks/upgrade.yml

```diff
@@ -44,6 +44,21 @@
     steps:
       - description: "Upgrading host: {{ item.name }}"
 
+- name: Gather self-heal facts about all gluster hosts in the cluster
+  # TODO: retry and delay removed because when added deligate_to it is not working as expected
+  gluster_heal_info:
+    name: "{{ volume_item.name }}"
+    status_filter: self-heal
+  register: self_heal_status
+  until: "self_heal_status | ovirtselfheal == False"
+  delegate_to: "{{ ovirt_hosts[0].name }}"
+  with_items:
+    - "{{ volumes }}"
+  loop_control:
+    loop_var: volume_item
+  when:  ovirt_clusters[0].gluster_service | bool
```

Comment:
not sure if is necessary when `gluster_volumes` will be empty list task won't run